pydy/viz/shapes.py

```diff
@@ -216,7 +218,7 @@ class Cube(Shape):
 
     Parameters
     ==========
-    length: float or SymPy expression
+    length : float or SymPy expression
```

Comment:
I think they should be floats. If you pass in an int it should just convert it to floats. I think he doesn't worry about it here because it eventually gets written to json and consumed by Javascript which doesn't have the concept of float/int.